pages/index.tsx

```diff
@@ -144,77 +274,258 @@ export default function Home() {
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
       </Head>
-      <main>
-        <nav>
-          <Image alt="Prisma logo" src="/logo.svg" width={105} height={32} />
-          <span className="badge">Early Access</span>
+      <main className={styles.main}>
+        <div className={styles.nav}>
+          <a href="https://www.prisma.io/">
+            <Image alt="Prisma logo" src="/logo.svg" width={105} height={32} />
+          </a>
           <span style={{ flex: 1 }}></span>
           <a
-            href="https://www.prisma.io/data-platform/accelerate"
+            href="https://github.com/prisma/accelerate-speed-test"
             target="_blank"
+            className={styles.githubIcon}
             rel="noopener noreferrer"
           >
-            <Button variant="secondary">Join the waitlist</Button>
+            <img src={"./github.svg"} width="32px" height="32px" />
           </a>
-        </nav>
-        <header>
-          <Title titleProps={{ style: { fontSize: `2rem` } }}>
-            Accelerate Speed test
-          </Title>
-          <p>
-            The test will run for ~5 seconds. See how many requests Accelerate
-            can process sequentially over that time.
+          <WebsiteButton
+            href="https://www.prisma.io/data-platform/accelerate"
```

Comment:
```suggestion
            href="https://prisma-data.typeform.com/to/WwPDKEQ5"
```
Should we link the typeform directly here?

Reply:
Agree with Tatiana, the 'Join the waitlist' button should take  you to the typeform